src/room/useGroupCall.ts

```diff
@@ -395,6 +403,43 @@ export function useGroupCall(groupCall: GroupCall): UseGroupCallReturnType {
     }
   }, [t]);
 
+  useEffect(() => {
+    const keyDownListener = (event) => {
+      // Check if keyboard shortcuts are enabled
+      const keyboardShortcuts = getSetting("keyboard-shortcuts", true);
+      if (!keyboardShortcuts) {
+        return;
+      }
+
+      if (event.key === "m") {
+        toggleMicrophoneMuted();
+      }
+      if (event.key === " ") {
+        setMicrophoneMuted(false);
+      }
```

Comment:
We might also want to think about adding 'v' if we're aiming for Jitsi parity (ie. camera mute toggle)